### PR TITLE
A very quick reread of the Form Login article

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: cookbook/security/form_login_setup.rst [hunk 1]
```diff
@@ -76,20 +76,15 @@ First, enable form login under your firewall:
...
 
     class SecurityController extends Controller
     {
-        public function loginAction(Request $request)
-        {
-            // todo...
-        }
     }
 
 Next, create two routes: one for each of the paths your configured earlier
```
Comment (xabbuh) [inline cookbook/security/form_login_setup.rst]: you

File: cookbook/security/form_login_setup.rst [hunk 2]
```diff
@@ -118,6 +114,8 @@ under your ``form_login`` configuration (``/login`` and ``/login_check``):
              */
             public function loginCheckAction()
             {
+                // this controller will not be executed,
+                // as the route is handled by the Security system
             }
```
Comment (xabbuh) [inline cookbook/security/form_login_setup.rst]: By the way, is this method really needed at all?

**Comment by weaverryan** [comment]
This kind if stuff is SUCH a big win. Docs "degrade" in their consistency over time. Thanks a lot of this :).

File: cookbook/security/form_login_setup.rst (hunk 1 above)
Comment (weaverryan) [inline cookbook/security/form_login_setup.rst]: nice catch - got it at sha: e2453038e4ea95af93ede147de0cbcc37e11359c

**Pull request merged**